### PR TITLE
Russell - L252.Trn_Inc_Elas_scenarios

### PR DESCRIPTION
**Pull request by russellhz**
This is basically the same exact code as L242.Bld.IncElas

#292

### CODE REVIEW

File: R/zchunk_L252.Trn_Inc_Elas_scenarios.R [hunk 1]
```diff
@@ -1,88 +1,161 @@
...
+
+module_socioeconomics_L252.Trn_Inc_Elas_scenarios <- function(command, ...) {
   if(command == driver.DECLARE_INPUTS) {
     return(c(FILE = "common/GCAM_region_names",
-             FILE = "energy/A52.demand",
              FILE = "socioeconomics/A52.inc_elas",
              "L101.Pop_thous_GCAM3_R_Y",
-             "L102.gdp_mil90usd_GCAM3_R_Y",
-             "L102.pcgdp_thous90USD_Scen_R_Y"))
+             FILE = "temp-data-inject/L102.gdp_mil90usd_GCAM3_R_Y",
```
Comment (d3y419) [inline R/zchunk_L252.Trn_Inc_Elas_scenarios.R]: tagging for GCAM3 legacy. no immediate action needed.

**Comment by rplzzz** [comment]
@d3y419 If the level2 ID string is missing, then the module isn't correct, and the merge shouldn't be approved.

**Comment by russellhz** [comment]
What does it mean to add level2 ID strings?
Just as a comment?

**Comment by rplzzz** [comment]
@russellhz This is being discussed over at #321.  It sound like we haven't fully decided how we are doing this.  I'll post some relevant questions so we can get some definitive answers.

**Comment by bpbond** [comment]
@russellhz Yeah, this isn't totally finalized. Now that Pralit is back from his break, we'll do so. I don't know that needs to hold up merging this, as long as you're willing to re-visit later and make final tweaks.

**Comment by russellhz** [comment]
@bpbond Sounds good!

**Comment by d3y419** [comment]
@rplzzz : approved?

**Comment by d3y419** [comment]
@rplzzz : approved?

**Comment by bpbond** [comment]
Note - Level 2 ID strings not an issue here (another chunk will handle XML generation; see #358 ). I will add required files to this PR once we approve and merge #358 .

**Pull request merged**